students/hattie2/Session01/break_me.py

```diff
@@ -0,0 +1,18 @@
+def makeNameError(): #makes a name error
+    x = y
+
+def makeTypeError():  # makes a type error
+    aString  = "Hello"
+    newstring = aString+1
+
+#def makeSyntaxError():  # makes a Syntax error
+#    aString = "Hello
+
+def makeAttributeError():  # makes an Attribute error
+    aString = "Hello"
+    newString = aString.value
+
+#makeNameError()
+#makeTypeError()
+#makeAttributeError()
+
```

Comment:
You might also want to look into python built in exceptions to make life easier.